qir/qat/Apps/Qat/ProfileConfiguration.hpp

```diff
@@ -0,0 +1,305 @@
+
+    configuration_manager.updateParameter("separate-circuits", true);
+
+    configuration_manager.updateParameter("allow-internal-calls", false);
+    configuration_manager.updateParameter("requires-qubits", true);
+    configuration_manager.updateParameter("requires-results", true);
+    configuration_manager.updateParameter("allow-poison", true);
+    configuration_manager.updateParameter("allow-undef", true);
+}
+
+inline void configureProviderb340Profile(ConfigurationManager& configuration_manager)
+{
+    // Setting profile validation configuration
+    configuration_manager.addConfig<ValidationPassConfiguration>(
+        "validation-configuration", ValidationPassConfiguration::fromProfileName("provider_b340"));
+
+    // Transformation
+    configuration_manager.disableSectionById("grouping");
+
+    configuration_manager.updateParameter("apply", true);
+    configuration_manager.updateParameter("validate", true);
+    configuration_manager.updateParameter("S", false);
+    configuration_manager.updateParameter("O0", false);
+    configuration_manager.updateParameter("O1", false);
+    configuration_manager.updateParameter("O2", false);
+    configuration_manager.updateParameter("O3", false);
+    configuration_manager.updateParameter("verify-module", true);
+    configuration_manager.updateParameter("experimental", false);
+    configuration_manager.updateParameter("dump-config", true);
+    configuration_manager.updateParameter("add-ir-debug", false);
+    configuration_manager.updateParameter("strip-existing-dbg", false);
+
+    configuration_manager.updateParameter("optimize-result-one", true);
+    configuration_manager.updateParameter("optimize-result-zero", true);
+    configuration_manager.updateParameter("optimize-result-comparison", true);
+    configuration_manager.updateParameter("use-static-qubit-array-allocation", false);
+    configuration_manager.updateParameter("use-static-qubit-allocation", false);
+    configuration_manager.updateParameter("use-static-result-allocation", false);
+
+    configuration_manager.updateParameter("disable-reference-counting", false);
+    configuration_manager.updateParameter("disable-alias-counting", false);
+    configuration_manager.updateParameter("use-static-result-allocation", false);
+
+    configuration_manager.updateParameter("disable-reference-counting", true);
+    configuration_manager.updateParameter("disable-alias-counting", true);
+    configuration_manager.updateParameter("disable-string-support", true);
+    configuration_manager.updateParameter("disable-record-output-support", false);
+
+    configuration_manager.updateParameter("allow-internal-calls", false);
+    configuration_manager.updateParameter("requires-qubits", true);
+    configuration_manager.updateParameter("requires-results", true);
+    configuration_manager.updateParameter("allow-poison", true);
+    configuration_manager.updateParameter("allow-undef", true);
```

Comment:
It looks like this chunk got duplicated below for this profile.

Reply:
Yes, this needs to be fixed.